internal/acceptance/data_pipelines_test.go

```diff
@@ -0,0 +1,28 @@
+package acceptance
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"testing"
+)
+
+func TestAccDataSourcePipelines(t *testing.T) {
+	accountLevel(t, step{
+		Template: `
+		data "databricks_pipelines" "this" {
```

Comment:
can you create a pipeline - there might be a situation, where test workspace has no pipelines at all. e.g. acceptance tests must be self-contained.

Reply:
acceptance test updated to include pipeline creation